pkg/proxy/nftables/proxier.go

```diff
@@ -262,6 +268,7 @@ func NewProxier(ipFamily v1.IPFamily,
 		networkInterfacer:   proxyutil.RealNetwork{},
 		conntrackTCPLiberal: conntrackTCPLiberal,
 		staleChains:         make(map[string]time.Time),
+		numgen:              "inc",
```

Comment:
this is going to be a bit of bikeshedding, but random seems to be a better default since is the one used by cilium and kube-proxy iptables, that I are used more than ivps https://gist.github.com/aojea/5f82db3ba5f1efd59bb73f4b28614a6a

Reply:
I kind of agree. Random is random, but people expect too much of other schedulers, even "rr". Like the endless discussion of "initial distribution", mentioned for instance in https://github.com/kubernetes/kubernetes/issues/121591#issuecomment-1792306681. Even maglev-hashing doesn't work out-of-the-box in the way people expect.

Reply:
I made some experiments of my own with K8s load-balancing, but without any http considerations, only plain tcp connects, at: https://github.com/uablrek/xcluster-ovls/tree/main/k8s-load-balancing

The hardest part was to create the table with Unix/Linux commands, please see:
https://github.com/uablrek/xcluster-ovls/blob/510cca913b632ae0a8506c45bf69d92bc1e6373e/k8s-load-balancing/k8s-load-balancing.sh#L72

Reply:
ChatGPT thinks IPTables proxy uses round-robbin algorithm, directing new requests to next destination in the list of pods.  Maybe that is what people think in general - round-robbing rather than random 🤔 

https://chat.openai.com/share/20271990-883b-476b-b5b0-3c2d2b7efbaf

Reply:
I advocate for trying to keep the same behavior as in iptables, but @danwinship is leading this feature, let's see what he thinks